04_sessions.html

```diff
@@ -109,89 +109,36 @@
         <td></td>
       </tr>
     </table>
-    <p>The process for successfully creating a session MUST be:</p>
+    <p>The <a>remote end steps</a> for the <dfn>New Session</dfn> command are:</p>
     <ol>
-      <li>The local end creates a new <a>Capabilities</a> instance describing the desired capabilities
-      for the session. The <a>Capabilities</a> object MUST be defined but MAY be empty.</li>
-      <li>The local end MUST populate <var>parameters</var> with an object containing an
-      entry named "desiredCapabilities" with the value set to the <a>Capabilities</a> instance from the previous step. An optional
-      "requiredCapabilities" entry MAY also be created and populated with a <a>Capabilities</a> instance. The "sessionId" fields
-      SHOULD be left empty.</li>
-      <li>The remote end MUST examine the "requiredCapabilities" parameter if specified, SHOULD examine the "desiredCapabilities" parameter, and MUST create a new session matching as many of the <a>Capabilities</a> as
-      possible.
-      How the new session is created depends on the implementation of this specification.
-        <ul>
-          <li>If any of the "requiredCapabilities" cannot be fulfilled by the new session, the remote end MUST quit the session and return the
-            <code><a href="#status-session-not-created">session not created</a></code> error code. The <var>sessionId</var> MUST be left as the default value unless one was set in the initial request, in which case that value MUST be used. The error message SHOULD list all unmet required capabilities though only a single required capability MUST be given.</li>
-          <li>If a capability is listed in both the requiredCapabilities and desiredCapabilities, the value in the requiredCapabilities MUST be used.</li>
-          </ul>
+      <li>
+        <p>
+         Let <var>resultant capabilities</var> be the result of calling <a>process capabilities</a>
+         and use <var>capabilities</var> as an argument. If a <a>session not created</a> is raised
```

Comment:
Better wording would be: `with <var>capabilities</var> as the argument.`

Also I think it would be better to say `if the result is an <a>error</a>, return an <a>error</a> with the code <a>session not created</a>.`

(See how [Get](https://w3c.github.io/webdriver/webdriver-spec.html#get) does it.)